ibis/backends/bigquery/__init__.py

```diff
@@ -558,7 +558,7 @@ def table(
         return rename_partitioned_column(table_expr, bq_table, self.partition_column)
 
     def _make_session(self) -> tuple[str, str]:
-        if self._session_dataset is None:
+        if self._session_dataset is None and hasattr(self, "client"):
```

Comment:
Might as well pull the client out using `getattr` since we're immediately using it.